meshes/collada_files/cf2_assembly.dae

```diff
@@ -1,40 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--
```

Comment:
We should try to leave the bitcraze note here in case anyone uses this collada file. You can also add an update note to it as well in the comment line like this:

- 2024-08 by R Bouwmeester, move model to have origin in center
- 2022-02 by K McGuire, initial collada file